src/main/cpp/t03_twos.cpp

```diff
@@ -23,5 +23,28 @@
 using namespace std;
 
 int t03_twos() {
+	int n;
+	cin >> n;
+	int s = 2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 's' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912)